service/controller/remotewrite/controller.go

```diff
@@ -0,0 +1,80 @@
+package remotewrite
+
+import (
+	"github.com/giantswarm/k8sclient/v7/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/v7/pkg/controller"
+	promclient "github.com/prometheus-operator/prometheus-operator/pkg/client/versioned"
+	vpa_clientset "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/client/clientset/versioned"
+
+	"github.com/giantswarm/prometheus-meta-operator/pkg/project"
+)
+
+type ControllerConfig struct {
+	K8sClient        k8sclient.Interface
+	Logger           micrologger.Logger
```

Comment:
Can you skim this down to just the 3 first ? You will not use most of the rest if any

Reply:
Sure! I wasn't sure about those attributes.

Reply:
Thanks :)